packages/fast-components-react-msft/src/button/button.tsx

```diff
@@ -28,7 +28,7 @@ class Button extends Foundation<IButtonHandledProps & IManagedClasses<IMSFTButto
                 {...this.unhandledProps()}
                 className={this.generateClassNames()}
                 managedClasses={this.props.managedClasses}
-                href={this.props.href}
+                href={this.props.href || null}
```

Comment:
I don't think you need to pass null here - the `BaseButton` should handle the `undefined` consistently

Reply:
updated (with force...sorry)